src/Symfony/Component/Translation/Tests/Formatter/IntlMessageFormatterTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Tests\Formatter;
+
+use Symfony\Component\Translation\Formatter\IntlMessageFormatter;
+
+class IntlMessageFormatterTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        if (!extension_loaded('intl')) {
+            $this->markTestSkipped(
+              'The Intl extension is not available.'
+            );
+        }
+    }
+
+    /**
+     * @dataProvider provideDataForFormat
+     */
+    public function testFormat($expected, $message, $number, $arguments)
+    {
+        $formatter = new IntlMessageFormatter();
+
+        $this->assertEquals($expected, trim($formatter->format('en', $message, $number, $arguments)));
+    }
+
+    public function testFormatWithNamedArguments()
+    {
+        if (version_compare(PHP_VERSION, '5.5.0', '<') || version_compare(INTL_ICU_VERSION, '4.8', '<')) {
```

Comment:
why is there a check on PHP 5.5 ? the skipping message only talks about ICU.

and PHP version checks should use ``PHP_VERSION_ID`` in Symfony